specification/timeseriesinsights/resource-manager/Microsoft.TimeSeriesInsights/stable/2020-05-15/timeseriesinsights.json

```diff
@@ -57,6 +57,12 @@
             "schema": {
               "$ref": "#/definitions/OperationListResult"
             }
+          },
+          "default": {
+            "description": "HTTP 404 (Not Found): The list of operations could not be found.",
```

Comment:
nit, but I usually see a generic error message here such as: "Error response describing why the operation failed." although I also noticed other services mentioning 404 as part of the description. Will check if there's any guidance around unifying these descriptions, but nothing to block this PR.

Reply:
Thanks, @jorgecotillo . I'd appreciate any guidance on this. If you find anything, could you mail me at my MSFT mail? 

Thanks!